src/Learners/ClassifierModels/ClassifierModels.csproj

```diff
@@ -14,7 +14,7 @@
   <Choose>
     <When Condition="'$(Configuration)'=='DebugFull' OR '$(Configuration)'=='ReleaseFull'">
       <PropertyGroup>
-        <TargetFramework>net462</TargetFramework>
+        <TargetFramework>net6.0-windows</TargetFramework>
```

Comment:
Is this a mistake?

Reply:
This is a policy decision -- going forward which version of .NET do we regard as the "full" version? When I wrote this, it made sense to me that it would be net6.0-windows (as I read "full" as "everything included", that is, will nothing left out to make it cross-platform); but if "full" is intended to mean "(non-Core/pre-.NET 5.0) .NET Framework" then I will change this. Please let me know.

Reply:
"full" is intended to mean "(non-Core/pre-.NET 5.0) .NET Framework"

Reply:
Thanks, understood. I've changed it.